cli/pkg/utils/utils.go

```diff
@@ -225,4 +228,24 @@ func ConvertMapI2MapS(v interface{}) interface{} {
 	}
 
 	return v
-}
\ No newline at end of file
+}
+
+func GetTimescaleDBURI(namespace, name string) (string, error) {
+	dbUriNotFoundError := "secrets \"" + name + "-timescaledb-uri\" not found"
+	dbUriError := errors.New(dbUriNotFoundError)
```

Comment:
is there really no better way to catch this? Can we use a list interface or something?